frontend/src/views/SettingWorkspaceGeneral.vue

```diff
@@ -26,7 +26,11 @@
           :disabled="!allowEdit"
         />
         <div for="databaseUrl" class="mt-2 textinfolabel">
-          {{ $t("settings.workspace.url-tip").replace('%s', placeholder) }}
+          <i18n-t keypath="settings.workspace.url-tip" tag="p">
```

Comment:
Is there a particular reason to choose tag="p" here?

According to https://vue-i18n.intlify.dev/guide/advanced/component.html#basic-usage

"You can choose the root node element type by specifying a tag prop. If omitted, it defaults to Fragments."

IIUC, we are adding an extra <p> with the change?

Reply:
Good point. It's unnecessary to add another DOM here. Will remove it